### PR TITLE
{data}[foss/2015a] GDAL 1.11.5 + 2.1.0

### PR DESCRIPTION
**Pull request by valtandor**
New EasyConfigs for updated versions of GDAL 1.11 and 2. Requires #4011.

### CODE REVIEW

**Comment by boegel** [comment]
@valtandor missing easyconfig for `PostgreSQL/9.5.3-foss-2015a`, do you have that available in another PR? OK to include it in here afaic.

File: easybuild/easyconfigs/g/GDAL/GDAL-1.11.5-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ('netCDF', '4.3.3.1'),
+    ('libspatialite', '4.3.0a'),
+    ('libpng', '1.6.20'),
+    ('expat', '2.1.0'),
+    ('PCRE', '8.37'),        # could probably be via sqlite, but currently not.
+#    ('sqlite', )            # via libspatialite
+#    ('libxml2', '2.9.3'),   # via libspatialite
+#    ('GEOS', '3.5.0'),      # via libspatialite
+#    ('HDF5', '1.8.16'),     # via netCDF and KEALib
+#    ('zlib', '1.2.8'),      # via HDF5 and libpng
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-1.11.5-foss-2015a.eb]: @valtandor is it worth keeping these in?

Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-1.11.5-foss-2015a.eb]: maybe it makes more sense to include a comment next to libspatialite mentioning it also pulls in SQLite, libxml2 and GEOS?

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.0'),
+    ('Perl', '5.22.0'),
+]
+
+configopts = '--with-openssl'
+
+# Client side only
+preinstallopts = 'make -C src/bin install && make -C src/include install && make -C src/interfaces install #'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb]: @valtandor what's this about?

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('zlib', '1.2.8'),
+]
+
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.0'),
+    ('Perl', '5.22.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb]: @valtandor easyconfig to resolve this dep is also missing... I'd say we should take care of that in a separate PR?

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb (hunk 2 above)
Comment (valtandor) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb]: @boegel This EasyConfig was prepared to install only the PostgreSQL client, not the database server  (see also https://www.postgresql.org/docs/current/static/install-procedure.html, having a look for "Client-only installation"). While we only need the client locally, I'm happy to take out these preinstallopts if the full server deployment is desired for the purposes of shared EasyConfigs.

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb (hunk 3 above)
Comment (valtandor) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb]: Probably a good thing.

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.3-foss-2015a.eb]: @valtandor We've installed the whole shabang up until now, so I'd stick to doing that; or introduce a `-client` `versionsuffix` in here.

If you go with the latter, include the URL as a comment above this `preinstallopts` line.

**Comment by boegel** [comment]
@valtandor I retriggered the tests after merging #4011, there's still a conflict on the PROJ version...

**Comment by valtandor** [comment]
@boegel And on the libreadline version (via SQLite). I'm trying to resolve it.

**Comment by verdurin** [comment]
@valtandor Please fix these style issues:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/GDAL/GDAL-2.1.3-gimkl-2017a-GEOS-3.5.1-Python-2.7.13.eb:32:60: E261 at least two spaces before inline comment
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/GDAL/GDAL-2.1.3-gimkl-2017a-GEOS-3.5.1-Python-2.7.13.eb:35:11: E221 multiple spaces before operator
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/GDAL/GDAL-2.1.3-gimkl-2017a-GEOS-3.5.1-Python-3.6.0.eb:32:60: E261 at least two spaces before inline comment
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/g/GDAL/GDAL-2.1.3-gimkl-2017a-GEOS-3.5.1-Python-3.6.0.eb:35:11: E221 multiple spaces before operator
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/p/PROJ/PROJ-4.9.2-foss-2015a.eb:25:69: W299 trailing whitespace
```

**Comment by migueldiascosta** [comment]
@valtandor, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**